src/analytics/parameter.ts

```diff
@@ -615,6 +617,18 @@ export function createStackParameters(scope: Construct): {
     default: 365,
   });
 
+  const dataFreshnessInHourParam = new CfnParameter(scope, 'DataFreshnessInHour', {
```

Comment:
Why do we need this one?

If it's required, create a share method with data processing.

Reply:
Need this parameter to get how many days data should be refreshed.
Added a common method.